src/pages/Subscribe/Sub_Info/SubscribeInfo.js

```diff
@@ -1,7 +1,155 @@
 import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
 
 const SubscribeInfo = () => {
-  return <h1>SubscribeInfo</h1>;
+  return (
+    <div>
+      <SubscribeInfoMain>
+        <HeaderFirstTitle>한 달에 한 번, 찾아오는 인생술</HeaderFirstTitle>
+        <HeaderSecondTitle>궁금한 담아박스를 구독해 보세요</HeaderSecondTitle>
+
+        <DamhwaBoxImg
+          src="../images/PhotoRoom-20221228_103237.png"
+          alt="담아박스 사진"
+        />
+      </SubscribeInfoMain>
+      <CarouselMain>
+        <CarouselBox>
+          <CarouselTitle>정통주 구독을</CarouselTitle>
+          <CarouselTitle>술담아로 시작해야 하는 이유</CarouselTitle>
+        </CarouselBox>
+        <CarouselImage
+          src="../images/PhotoRoom-20221228_103237.png"
+          alt="담아박스 사진"
+        />
+        <BottomGoDamhwaBox>
+          <BottomGoDamhwaBoxTitle>
+            자, 이제 나만의 '인생술 찾기 여정'을
+          </BottomGoDamhwaBoxTitle>
+          <BottomGoDamhwaBoxTitle>시작할 준비 되셨나요?</BottomGoDamhwaBoxTitle>
+          <BottomCalenderBox>
+            <CalenderBoxTitle>담아박스는</CalenderBoxTitle>
+            <CalenderBoxTitleBold>
+              이번 달 둘째 주 목요일에 도착해요.
+            </CalenderBoxTitleBold>
+            <CalenderBoxTitleBoldDate>
+              1월 결제일과 배송일은?
+            </CalenderBoxTitleBoldDate>
+            <BottomCalender
+              src="../images/tOC2-1671419966955-0.png"
+              alt="달력사진"
+            />
+          </BottomCalenderBox>
+          <GoDamhwaButtonBox>
+            <Link to="/Subscribe">
+              <GoDamhwaButton>구독하러 가기 ></GoDamhwaButton>
+            </Link>
+          </GoDamhwaButtonBox>
+        </BottomGoDamhwaBox>
+      </CarouselMain>
+    </div>
+  );
 };
 
+const SubscribeInfoMain = styled.div`
+  background-color: rgb(247, 247, 247);
+  text-align: center;
+`;
+
+const HeaderFirstTitle = styled.div`
+  padding-top: 50px;
+  font-size: 24px;
+`;
+
+const HeaderSecondTitle = styled.div`
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 1.5715;
+`;
+const DamhwaBoxImg = styled.img`
+  width: 364px;
+  border-radius: 15px;
+  margin: 50px 0px 100px 0px;
+`;
+
+const CarouselMain = styled.div`
+  margin: auto;
+  width: 100%;
+`;
+
+const CarouselBox = styled.div`
+  margin: 20px 0px;
+`;
+
+const CarouselTitle = styled.p`
+  color: #0097f3;
+  font-weight: bold;
+  font-size: 28px;
+  text-align: center;
+  line-height: 1.5715;
+`;
+
+const CarouselImage = styled.img`
+  width: 375px;
+  background-color: blue;
+  display: flex;
+  margin: 10px auto 0px auto;
+  justify-content: center;
```

Comment:
```suggestion
  display: flex;
   justify-content: center;
  width: 375px;
  margin: 10px auto 0px auto;
  background-color: blue;
```

이 순서대로 CSS 속성순서 변경 해 주시고,
이 기준은
https://study.wecode.co.kr/session/content/193
여기서 4번 CSS 속성순서에서 확인 해 주세요~

큰~ 단위에서
작은~~ 단위로 순서 정렬 해 주시면 됩니다.

다른 코드도 순서대로 정렬 잘 되어있는지 확인 해 주시고, 안되있다면 정렬 해 주세요!